microsoftgraph/microsoft-graph-explorer-v2
# Bug/shows error response in the response window

## Overview

Displays the error response return when a query fails
Also displays the error response when someone views the history 
Localizes the word milliseconds

Fixes #255 

### Demo

![image](https://user-images.githubusercontent.com/53429535/67852828-364b3580-fb1e-11e9-9211-ce3135444228.png)

## Testing Instructions

* Add something that does not exist eg 'a' to the end of the request url to trigger an error 404
* Notice the description provided on the response body when this fails
